package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "TuTiempo",
+  "version": "1.0.0",
+  "description": "TuTiempo is a Web Applciation made using AngularJS for displaying forecast predictions in desired places",
+  "main": "app.module.js",
+  "scripts": {
+    "dev": "webpack-dev-server --open --hot --mode development --config ./config/webpack.config.js",
+    "build": "webpack --progress --hide-modules --mode production --config ./config/webpack.config.js"
```

Comment:
You should make use of HtmlWebpackPlugin in order to produce the index.html to serve your bundle

Reply:
Solved on #1 . Ill merge that first, will be solved in this branch too